_posts/2019-12-19-sw-ix-review.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+title: Star Wars Episode IX The Rise of Skywalker Review [SPOILERS]
+date: 2019-12-19 16:28:05.000000000 +01:00
+type: post
+published: true
+status: publish
+categories: [General]
+image:
+image2:
+author: Valentino Urbano
+---
+
+[Obviously heavy spoilers for the movie and SW 1 to 9]
+
+Coming in I had really love expectations, unfortunately that was not enough.
+
+The movie at the same level of shittiness as [The Force Awakens][1] and way worse than The Last Jedi. It seems like Disney doesn't care at all about existing lore.
+
+They made a plot and rammed it through not caring about existing characters arcs or existing lore. I see a lot of similarities to Game Of Thrones Season 8 in there, but it is not as bad as GoT in my opinion.
+
+Where to start... so many things wrong with this movie.
+
+## The whole SW story arc is nullified
+
+George Lucas has said repeatedly that the whole arc of SW 1 to 6 was the fall to Anakyn to the dark side and its redemption and even if you don't like the prequels (I love them) you can see how they are consistent with the overarching story and rules and how they deliver on that vision. The whole reason for his fall was trying to save Padme. The new trilogy now make it seem that bringing someone back from death is a joke.
+
+The whole idea of the chosen one that would bring balance to the force that went vindicated in episode 6 is now in shambles. Anakyn in the end did not bring balance. He failed to kill the emperor. What a joke.
+
+## No bigger plot
+
+SW 7 to 9 don't have an overline structure. They just decided 3 plots for the 3 movies and that's it. They have characters in common, are set in the same universe, and they happen chronologically but apart from that there is not much else going on.
```

Comment:
Consistent structure?

Reply:
It is not that it was not consistent it was not there AT ALL. Way worse